pages/sending/input/beats.rst

```diff
@@ -0,0 +1,13 @@
+.. _beats_input:
+
+*****
+Beats  
+*****
+
+Beats are open source data shippers. They are single-purpose tools. 
+Some beats are created and maintained by the company elastic. Those listed in `this beats overview <https://www.elastic.co/guide/en/beats/libbeat/current/beats-reference.html>`__. The community creates an additional `wide range of beats <https://www.elastic.co/guide/en/beats/libbeat/current/community-beats.html>`__. Most of the beats should work out of the box with the Graylog beats input. But it might be needed to adjust the settings. 
```

Comment:
Capitalization change.

```suggestion
Some beats are created and maintained by the company Elastic. Those listed in `this beats overview <https://www.elastic.co/guide/en/beats/libbeat/current/beats-reference.html>`__. The community creates an additional `wide range of beats <https://www.elastic.co/guide/en/beats/libbeat/current/community-beats.html>`__. Most of the beats should work out of the box with the Graylog beats input. But it might be needed to adjust the settings. 
```